### ISSUE TITLE
Accuracy 

### ISSUE THREAD

**Issue by amna-ai** [dxli94/WLASL#49]
When I tested the I3D model for 100 classes, I achieved the following accuracy:
top-k average per class acc: 0.670542635658915, 0.8923076923076927, 0.9346153846153845
0.67 for top 1 class
0.89 for top 5
0.93 for top 10

It is more than the ones mentioned in the title of pretrained weights?
Did I do something wrong? How can I get more than mentioned?

**Comment by XiongTLu** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    Based on the WLASL paper, it is recorded in Table 4 that the top-10 accuracy of the I3D model trained with WLASL100 is 89.92%, so I think your training result is also very good and reliable.About your higher accuracy, I think it's because your hardware is newer than the author's.
    
        
            
        
    
    
     
    
        
                    ***@***.***
        
    
 

    On 8/23/2022 ***@***.***> wrote： 



When I tested the I3D model for 100 classes, I achieved the following accuracy:
top-k average per class acc: 0.670542635658915, 0.8923076923076927, 0.9346153846153845
0.67 for top 1 class
0.89 for top 5
0.93 for top 10
It is more than the ones mentioned in the title of pretrained weights?
Did I do something wrong? How can I get more than mentioned?

**Comment by hshreeshail** [comment]
Yeah. I am also getting better than reported results, though not as good as yours:
Metric:      Top-1  Top-5  Top-10
Reported:  65.89  84.11   89.92
Mine____:         67.07  84.58   90.25
Yours____:        67.05  89.23   93.46

However, I am getting worse that reported results for 2000 classes
Reported:  32.48  57.31   66.31
Mine____:         30.45  55.36   63.87


Is this change simply down to hardware difference? Is it significant?

**Comment by amna-ai** [comment]
I don't know if hardware is causing this result. Did you find any other reason?

**Comment by hshreeshail** [comment]
No. Not sure what the reason for the discrepancy is.

**Comment by 781696880** [comment]
I am also getting better than reported results when testing I3D model on 100 classes, 
when i use the provide model FINAL_nslt_100_iters=896_top1=65.89_top5=84.11_top10=89.92.pt, I get the  result same with you:
top-1,5,10 average per class acc: 67.06, 84.58, 90.25
when i use own trained model nslt_100_006237_0.693798.pt (21epoch), I get the better result:
top-1,5,10 average per class acc: 70.55, 88.00, 94.25

Did you find the reason? Could you tell me, please? Thank you!
